tests/unit_tests/test_rating_service.py

```diff
@@ -343,3 +343,20 @@ async def test_game_rating_error_handled(rating_service, game_info, bad_game_inf
     service._logger.warning.assert_called()
     # second game: results have been saved.
     service._persist_rating_changes.assert_called_once()
+
+
+async def test_game_update_empty_resultset(rating_service):
+    service = rating_service
+    service._persist_rating_changes = CoroutineMock()
```

Comment:
You are mocking this, and directly calling it later... So this test doesn't do anything